diff --git a/bindings/python/CMakeLists.txt b/bindings/python/CMakeLists.txt
index 305629b..244ad14 100644
--- a/bindings/python/CMakeLists.txt
+++ b/bindings/python/CMakeLists.txt
@@ -12,7 +12,7 @@ enable_testing()
 
 add_subdirectory(tests)
 
-set(Python_ADDITIONAL_VERSIONS 3.6 3.5 2.7 2.6)
+set(Python_ADDITIONAL_VERSIONS 3.9)
 FIND_PACKAGE(PythonLibs)
 IF (PYTHON_LIBRARY)
   FIND_PACKAGE(PythonInterp REQUIRED)
diff --git a/bindings/python/pywsman/setup.py b/bindings/python/pywsman/setup.py
index e43f4dd..671f087 100644
--- a/bindings/python/pywsman/setup.py
+++ b/bindings/python/pywsman/setup.py
@@ -5,8 +5,9 @@ setup.py file for pywsman
 """
 
 from __future__ import print_function
-from distutils.core import setup, Extension
 from distutils.command.build import build as _build
+from setuptools import setup, Extension
+
 import subprocess
 import shutil
 import sys
diff --git a/debian/changelog b/debian/changelog
index 8ed2f43..46eeda1 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+openwsman (2.6.5-0ubuntu7) jammy; urgency=medium
+
+  * Change build to python3
+  * Makes the python version clear in the package name
+
+ -- Joshua Kugler <joshua@metify.io>  Thu, 26 May 2022 16:09:00 -0800
+
 openwsman (2.6.5-0ubuntu6) jammy; urgency=medium
 
   * No-change rebuild against libssl3
diff --git a/debian/control b/debian/control
index 4b77b4a..5ef07c6 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: openwsman
 Section: admin
 Priority: extra
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
-Build-Depends: debhelper (>= 9.0.0), cmake (>= 2.8.5), libssl-dev, libpam0g-dev, libxml2-dev, libcurl4-openssl-dev, libcimcclient0-dev, swig, python2-dev, dh-python
+Build-Depends: debhelper (>= 9.0.0), cmake (>= 2.8.5), libssl-dev, libpam0g-dev, libxml2-dev, libcurl4-openssl-dev, libcimcclient0-dev, swig, python3-dev, dh-python
 Standards-Version: 3.9.5
 Homepage: http://openwsman.github.io/
 
@@ -137,10 +137,10 @@ Description: Open Web Services Manager C++ bindings development files
  This package provides the development files for the C++ interface to the
  Openwsman client libraries
 
-Package: python-openwsman
-Section: python
+Package: python3-openwsman
+Section: python3
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libwsman1 (= ${binary:Version}), ${python:Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}, libwsman1 (= ${binary:Version}), ${python3:Depends}
 Description: Open Web Services Manager Python bindings
  Openwsman is a project intended to provide an open-source implementation of
  the Web Services Management specification (WS-Management) and to expose
diff --git a/debian/python-openwsman.install b/debian/python-openwsman.install
deleted file mode 100644
index 46ece99..0000000
--- a/debian/python-openwsman.install
+++ /dev/null
@@ -1,2 +0,0 @@
-usr/lib/python*/dist-packages/_pywsman.so
-usr/lib/python*/dist-packages/pywsman.py
diff --git a/debian/python3-openwsman.install b/debian/python3-openwsman.install
new file mode 100644
index 0000000..7b71561
--- /dev/null
+++ b/debian/python3-openwsman.install
@@ -0,0 +1,2 @@
+usr/lib/python3*/dist-packages/_pywsman.so
+usr/lib/python3*/dist-packages/pywsman.py
diff --git a/debian/rules b/debian/rules
index 7b6eea9..c0e6112 100755
--- a/debian/rules
+++ b/debian/rules
@@ -8,7 +8,7 @@ export CXXFLAGS=$(shell dpkg-buildflags --get CXXFLAGS)
 export DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
 %:
-	dh $@ --builddirectory=build --with python2
+	dh $@ --builddirectory=build --with python3
 
 override_dh_auto_configure: 
 	mkdir build
